src/Bicep.Core.Samples/Files/InvalidLoadFunctions_CRLF/main.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
>$schema [](http://example.com/codeflow?start=3&length=7)

This file doesn't match the source. Missing baseline update?

Reply:
main.json shouldn't be for invalid cases. I did a folder copy from the valid case - hence the problem. deleted now.